tensorflow/inference/docker/build_artifacts/sagemaker/nginx.conf.template

```diff
@@ -15,7 +15,7 @@ http {
   include /etc/nginx/mime.types;
   default_type application/json;
   access_log /dev/stdout combined;
-  js_include tensorflow-serving.js;
+  js_import tensorflowserving.js;
```

Comment:
Let's use the same files from the repo 
https://github.com/aws/sagemaker-tensorflow-serving-container/blob/master/docker/build_artifacts/sagemaker/nginx.conf.template#L18